test/load_data.jl

```diff
@@ -0,0 +1,21 @@
+
+@testset "load_timeseries_data" begin
+    # load one and check if all the fields are correct
+    ts_input_data = load_timeseries_data(:CEP_GER1)
+    @test ts_input_data.K == 366
+    @test ts_input_data.T == 24
+    @test all(ts_input_data.weights .== 1.)
+
+    # load all four by name - just let them run and see if they run without error
+    ts_input_data = load_timeseries_data(:DAM_GER)
```

Comment:
I like the usage of a Symbol there. I think I will do the same in CapExp

Reply:
Sounds good. I liked the idea of having a different type for given input, having two strings could be confusing to the user.